Source/Public/Publish-Bicep.ps1

```diff
@@ -0,0 +1,46 @@
+function Publish-Bicep {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory, Position = 1)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Path,
+
+        [Parameter(Mandatory, Position = 2)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Target          
+    )
+    begin {
+        # Check if a newer version of the module is published
+        if (-not $Script:ModuleVersionChecked) {
+            TestModuleVersion
+        }       
+    }
+
+    process {
+        $BicepFile = Get-Childitem -Path $Path -File
+        $LoginServer = (($target -split ":")[1] -split "/")[0]
```

Comment:
I find this very misleading, We separate the login server here, but the only place we use it is in a verbose message stating "login successful", however there is no specific login done in this file. If login is performed inside `Publish-BicepNetFile` then that CmdLet, not this one, should handle this verbose output.

I vote to remove this part completely and leave login logging to wherever login is performed.